server/src/events.mjs

```diff
@@ -92,10 +93,79 @@ function sendEventListenerAck(ws, oMsg) {
   logger.debug(`Sent event listener ack message: ${ackMessage}`);
 }
 
+// sendEvent function to handle pre- post- method/event triggers
```

Comment:
I think this comment should just be something like: send event, including pre- and post-event trigger processing

(because it is also used when the REST API or CLI is used to send an event; it's not _just_ for ctx.sendEvent() calls in pre-/post- triggers)

Reply:
modified the comment